pkg/apis/migration/v1alpha1/migplan_types.go

```diff
@@ -373,6 +382,22 @@ func (r *MigPlan) UpdateRegistryDeployment(storage *MigStorage, deployment *apps
 								Protocol:      kapi.ProtocolTCP,
 							},
 						},
+						LivenessProbe: &kapi.Probe{
+							Handler: kapi.Handler{
+								HTTPGet: &kapi.HTTPGetAction{
+									Path: "/",
+									Port: intstr.IntOrString{IntVal: 5000},
+								},
+							},
+						},
+						ReadinessProbe: &kapi.Probe{
+							Handler: kapi.Handler{
+								HTTPGet: &kapi.HTTPGetAction{
+									Path: "/",
```

Comment:
see this comment https://github.com/konveyor/mig-controller/issues/626#issuecomment-686721794, specifically

```
# Note 2 : using path `:5001/debug/health` would be more 
#         informative to user if it outputs acceptable HTTP codes for probes to interpret.
```

change the path to /debug/health and port according to the following comment

Reply:
Updated the path